src/dlgConnectionProfiles.cpp

```diff
@@ -380,24 +399,16 @@ void dlgConnectionProfiles::slot_save_name()
         QDir currentPath(mudlet::getMudletPath(mudlet::profileHomePath, currentProfileEditName));
         QDir dir;
 
+        removeNotification(tr("Could not rename your profile data on the computer."));
+        removeNotification(tr("Could not create the new profile folder on your computer."));
         if (currentPath.exists()) {
             // CHECKME: previous code specified a path ending in a '/'
             QDir parentpath(mudlet::getMudletPath(mudlet::profilesPath));
             if (!parentpath.rename(currentProfileEditName, newProfileName)) {
-                notificationArea->show();
-                notificationAreaIconLabelWarning->show();
-                notificationAreaIconLabelError->hide();
-                notificationAreaIconLabelInformation->hide();
-                notificationAreaMessageBox->show();
-                notificationAreaMessageBox->setText(tr("Could not rename your profile data on the computer."));
+                addNotification(tr("Could not rename your profile data on the computer."));
             }
         } else if (!dir.mkpath(mudlet::getMudletPath(mudlet::profileHomePath, newProfileName))) {
-            notificationArea->show();
-            notificationAreaIconLabelWarning->show();
-            notificationAreaIconLabelError->hide();
-            notificationAreaIconLabelInformation->hide();
-            notificationAreaMessageBox->show();
-            notificationAreaMessageBox->setText(tr("Could not create the new profile folder on your computer."));
+            addNotification(tr("Could not create the new profile folder on your computer."));
```

Comment:
:thinking: Would it be helpful to list the full, absolute path and folder name here so the user knows exactly where the new folder **hasn't** been created?